tests/library/test_metrics.py

```diff
@@ -1432,3 +1433,35 @@ def test_llm_as_judge_metric(self):
         ]
 
         self.assertListEqual(actual_scores, expected_scores)
+
+
+def test_fuzzyner(self):
+    metric = FuzzyNer()
+    predictions = [
+        [
+            ("jar htaras", "Person"),
+            ("Marathahalli", "Location"),
+            ("IBM", "Org"),
+        ]
+    ]
+    references = [
+        [
+            [
+                ("jar htaras", "Person"),
+                ("Marathahalli ring road", "Location"),
+            ]
+        ]
+    ]
+    outputs = apply_metric(
+        metric=metric, predictions=predictions, references=references
+    )
+    global_target = 1.0
+    self.assertAlmostEqual(global_target, outputs[0]["score"]["global"]["f1_Person"])
+    global_target = 0.0
```

Comment:
Would Ner() metric return a different result?

Reply:
we have changed the predicton and reference inputs

Reply:
But NER would match "jar htaras" to "jar htaras" and not match  "Marathahalli" "Marathahalli ring road", exactly as the fuzzy match does.   I would expect the test to match   "Marathahalli" and "Marathahalli ring road".

Reply:
Tests added.